### DISCUSSION TITLE
Spec: Jan Themes

### DISCUSSION THREAD

**Discussion by dan-menlo** [janhq/jan#3566] [Architecture Specs]
## Overview

- Jan should support Themes
- Similar to https://code.visualstudio.com/docs/getstarted/themes
- Similar to https://help.obsidian.md/Extending+Obsidian/Themes

## Initial Thoughts

- Can themes should be their own repo, e.g. https://github.com/primer/github-vscode-theme (i.e. let community build their own)
- Jan should expose a clear Theme API, inspired by VSCode or Obsidian

## Theme API Inspirations

- https://docs.obsidian.md/Themes/App+themes/Build+a+theme
- https://code.visualstudio.com/api/extension-guides/color-theme

**Comment by urmauur** [comment]
@dan-homebrew thanks for bring this up, i'll figured out and make a plan first then back to u
